src/js/components/Pagination/Pagination.js

```diff
@@ -176,9 +178,41 @@ const Pagination = forwardRef(
       ...navProps[control],
     }));
 
+    const itemsBegin = step * (activePage - 1) + 1;
+    const itemsEnd = itemsBegin + step - 1;
+    let showSummary;
+    if (showSummaryProp === true)
+      showSummary = (
+        <Text>
+          Showing {itemsBegin} - {itemsEnd} of {numberItems}
+        </Text>
+      );
+    else showSummary = showSummaryProp;
+
+    // only apply these if the caller hasn't already specified their own theming
+    // otherwise, let the caller control
+    const showSummaryStyles =
```

Comment:
By the name, I was expecting this style the summary. This seems like it is more about styling the surrounding container, as opposed to styling the summary itself.

Reply:
You're right, isn't not styling the summary but instead is styling the container. These are the defaults to set if the summary is being displayed. Open to changing to `containerProps` or something else or (like your comment below) looking for a way to add them as defaults in the theme.